models/src/constants.rs

```diff
@@ -39,7 +39,7 @@ pub const PUBLIC_KEY_NAME: &str = "tls.crt";
 pub const PRIVATE_KEY_NAME: &str = "tls.key";
 pub const TLS_KEY_MOUNT_PATH: &str = "/etc/brupop-tls-keys";
 // Certificate object name
-pub const ROOT_CERTIFICATE_NAME: &str = "root-certificate";
+pub const ROOT_CERTIFICATE_NAME: &str = "brupop-selfsigned-ca";
```

Comment:
I think this is probably safe but I'm digging into how we use cert-manager under the hood to ensure we aren't going to break something else down the line. Do you happen to have links to what led you to this change?

Reply:
I do not unfortunately. I can just attest from what Jack mentions [here](https://github.com/bottlerocket-os/bottlerocket-update-operator/issues/486#issuecomment-1894696759) that this resolved an issue on our end.

Reply:
Since it might help you: I just fixed this bug and set the value to cert-manager.io/inject-ca-from: brupop-bottlerocket-aws/brupop-apiserver-certificate